### DISCUSSION TITLE
Add option to set a prefix URL

### DISCUSSION THREAD

**Discussion by qbixx** [spatie/statamic-responsive-images#176] [Ideas]
It would be nice if it would be possible to set a prefix to the image URL. This could become in handy when working with e.g. an Amazon S3 bucket or CDN.

**Comment by ncla** [comment]
You could feature request this perhaps on statamic/ideas or statamic/cms repositories. This addon utilizes core image service classes, from which these URLs are generated, as such I think it is more suitable to be implemented on CMS/core level itself.
